tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
+    "importHelpers": true,
```

Comment:
Some updated libraries bring `import * from 'tslib'`, so it's best to use tslib for ourselves as well to not bloat the bundle any further.